aevnmt/hparams/hparams.py

```diff
@@ -57,7 +57,8 @@ def make_grouped_parser(arg_groups, check_required=True):
     :param check_required: Check for required arguments, defaults to True
     :return: argument parser containing the args from arg_groups.
     """
-    parser = ArgumentParser(formatter_class='default_argparse')
+    #parser = ArgumentParser(formatter_class='default_argparse')
+    parser = ArgumentParser()
```

Comment:
Why is this change needed?

Reply:
I got the following error when trying to train a model under Python 3.6.9, and argparse version 1.1:

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/argparse.py", line 1357, in add_argument
    self._get_formatter()._format_args(action, None)
  File "/usr/lib/python3.6/argparse.py", line 2362, in _get_formatter
    return self.formatter_class(prog=self.prog)
TypeError: 'str' object is not callable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/vmsanchez/envs/aevnmt.pt/lib/python3.6/site-packages/aevnmt/train_monolingual.py", line 278, in <module>
    main()
  File "/home/vmsanchez/envs/aevnmt.pt/lib/python3.6/site-packages/aevnmt/train_monolingual.py", line 209, in main
    hparams = Hyperparameters()
  File "/home/vmsanchez/envs/aevnmt.pt/lib/python3.6/site-packages/aevnmt/hparams/hparams.py", line 13, in __init__
    self._parser = make_grouped_parser(arg_groups, check_required)
  File "/home/vmsanchez/envs/aevnmt.pt/lib/python3.6/site-packages/aevnmt/hparams/hparams.py", line 60, in make_grouped_parser
    parser = ArgumentParser(formatter_class='default_argparse')
  File "/home/vmsanchez/envs/aevnmt.pt/lib/python3.6/site-packages/jsonargparse/core.py", line 164, in __init__
    super().__init__(*args, **kwargs)
  File "/home/vmsanchez/envs/aevnmt.pt/lib/python3.6/site-packages/jsonargparse/core.py", line 69, in __init__
    super().__init__(*args, **kwargs)
  File "/usr/lib/python3.6/argparse.py", line 1662, in __init__
    help=_('show this help message and exit'))
  File "/home/vmsanchez/envs/aevnmt.pt/lib/python3.6/site-packages/jsonargparse/core.py", line 89, in add_argument
    action = super().add_argument(*args, **kwargs)
  File "/usr/lib/python3.6/argparse.py", line 1359, in add_argument
    raise ValueError("length of metavar tuple does not match nargs")
ValueError: length of metavar tuple does not match nargs
```

Reply:
Lets keep this change, it seems jsonargparse changed the formatter support. Feel free to merge

Reply:
Thanks!